multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py

```diff
@@ -344,7 +355,11 @@ def create_transaction_for_setting_special_role_on_non_fungible_token(
         add_role_nft_burn: bool,
         add_role_nft_update_attributes: bool,
         add_role_nft_add_uri: bool,
-        add_role_esdt_transfer_role: bool
+        add_role_esdt_transfer_role: bool,
+        add_role_esdt_modify_creator: bool = False,
+        add_role_nft_recreate: bool = False,
+        add_role_esdt_set_new_uri: bool = False,
+        add_role_esdt_modify_royalties: bool = False
```

Comment:
can we have all add_role parameters as optional with default values?

Reply:
they are set as `False` by default now.